cdk/README.md

```diff
@@ -1,6 +1,13 @@
-# Welcome to your CDK TypeScript project!
+# Getting Started
+1. Navigate to the cdk directory in terminal
+2. `npm install monocdk`
+3. Run `npm run build` and `cdk synth` to test that everything is setup correctly and test cdk changes
```

Comment:
This assumes that they install CDK globally. I think we should keep everything contained in NPM and just modify/add NPM scripts in `package.json` to run the required CDK commands